src/pages/Logout.js

```diff
@@ -58,6 +40,7 @@ const Logout = () => {
       console.log(response);
       setUpdateLoading(false);
       setMessage('Update successful');
+      setCurrentUser(response?.data?.user)
```

Comment:
This calls `setCurrentUser` state function inside `user_context.js` file (line 31). All components that are reading the `currentUser` state variable from `useUserContext` will receive this information.